src/app.rs

```diff
@@ -184,20 +184,20 @@ impl<'a, 'b> Application<'a, 'b> {
                         },
 
                         &WindowEvent::AxisMotion {
-                            device_id, axis, ..
+                            device_id, axis, value
                         } => for mut stored_event in &mut events {
                             if let &mut Event::WindowEvent {
```

Comment:
Maybe it's better to store different types of events separately, instead of `if let` each event?

Reply:
Store them in more vecs?  That's probably more inefficient given this vec's purpose is to be small.